utilities/generate_cluster_configs_from_vagrant_hosts.py

```diff
@@ -19,17 +19,21 @@ def generate_cluster_configs_from_vagrant(private_network, public_network):
 
     if private_network and public_network:
         raise ProvisioningError("Invalid private_network and public_network flags")
+    elif private_network and public_network_ethernet:
+        raise ProvisioningError("Invalid private_network and public_network_ethernet flags")
```

Comment:
Would public_network and public_network_ethernet still fall through?

I would check if one flag is set, the other two are not.